### ISSUE TITLE
Accessibility of some members of J2N.Collections.BitSet does not match Apache Harmony

### ISSUE THREAD

**Issue by NightOwl888** [NightOwl888/J2N#7]
Some members of `BitSet` were inadvertently made not inheritable, when they should be marked virtual to match the accessibility of Apache Harmony. These members are:

```
public bool Get(int position)
public BitSet Get(int position1, int position2)
public void Set(int position)
public void Set(int position, bool value)
public void Set(int position1, int position2)
public void Set(int position1, int position2, bool value)
public void Clear()
public void Clear(int position)
public void Clear(int position1, int position2)
public void Flip(int position)
public void Flip(int position1, int position2)
public bool Intersects(BitSet bitSet)
public void And(BitSet bitSet)
public void AndNot(BitSet bitSet)
public void Or(BitSet bitSet)
public void Xor(BitSet bitSet)
public int Count (This is for compatibility, we should have a member with the same value named Capacity, and this hidden from public view)
public int Length
```
